Documentation/ExtensionArchitecture/ExtendingTca/Examples/Index.rst

```diff
@@ -74,20 +74,24 @@ Here's the complete code, taken from file
    TCA reference on how to implement it: `t3tca:columns-user`.
 
 
-In the first statement the fields are added by calling
-:php:`ExtensionManagementUtility::addTCAcolumns()'. The first parameter is the
-name of the table to which the fields should be added. The second parameter is
-and array of the fields to be added. Each field is represented in the
-:ref:`TCA syntax for columns <t3tca:columns>`.
-
-In the second statement the fields are added to the "types" definition of the
-"fe\_users" table by calling
-:php:`ExtensionManagementUtility::addToAllTCAtypes()`. The first parameter is
-the table name once more. The second parameter is a string, in the syntax as
-used in the :ref:`showitem property of types in TCA
-<t3tca:types-properties-showitem>`. An optional third
-specify the position (:php:`'before'` or :php:`'after'`) in relation to an
-existing field. So you could call optionally::
+In this example the first method call adds fields using
+:php:`ExtensionManagementUtility::addTCAcolumns()`. Parameters:
```

Comment:
should we describe the outcome here (something like "their column configuration is now available, but they are not shown as they are not configured for a type" to make it clearer why/what the next step is about?

Reply:
Yes, good idea.